src/hotspot/cpu/arm/templateTable_arm.cpp

```diff
@@ -2608,6 +2608,68 @@ void TemplateTable::load_field_cp_cache_entry(Register Rcache,
   }
 }
 
+// The rmethod register is input and overwritten to be the adapter method for the
+// indy call. Link Register (lr) is set to the return address for the adapter and
+// an appendix may be pushed to the stack. Registers r0-r3 are clobbered
```

Comment:
"Registers r0-r3 are clobbered". This is a copy-paste error, it seems. Which registers this method really clobbers? R1..R3?

Reply:
Fixed comment, thanks!